templates/repo/blame.tmpl

```diff
@@ -1,3 +1,15 @@
+{{if or .UsesIgnoreRevs .FaultyIgnoreRevsFile}}
+	{{$revsFileLink := URLJoin .RepoLink "src" .BranchNameSubURL "/.git-blame-ignore-revs"}}
+	{{if .UsesIgnoreRevs}}
+		<div class="ui info message">
+			<p>{{.locale.Tr "repo.blame.ignore_revs" $revsFileLink | Str2html}}</p>
```

Comment:
Maybe it could add `?bypass-blame-ignore=true` link here, than no need to write the document, and it's more friendly to end users?